CHANGELOG.md

```diff
@@ -38,6 +38,9 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### Dependencies
 
+* [\#6193](https://github.com/cosmos/ibc-go/pull/6193) Bump Cosmos SDK to v0.50.6.
+* [\#6193](https://github.com/cosmos/ibc-go/pull/6193) Bump `cosmossdk.io/store` to v1.1.0.
```

Comment:
I added this but unsure how much granularity we need in the changelog for sdk deps. 

At some point we will probably need to track some of them more closely when the cosmos-sdk dep is phased out